src/Files.App/Actions/Open/EditInNotepadAction.cs

```diff
@@ -20,7 +20,9 @@ public RichGlyph Glyph
 
 		public bool IsExecutable =>
 			context.SelectedItem is not null &&
-			FileExtensionHelpers.IsBatchFile(context.SelectedItem.FileExtension);
+			(FileExtensionHelpers.IsBatchFile(context.SelectedItem.FileExtension) ||
+			FileExtensionHelpers.IsAhkFile(context.SelectedItem.FileExtension) ||
+			FileExtensionHelpers.IsTextFile(context.SelectedItem.FileExtension));
```

Comment:
```suggestion
			FileExtensionHelpers.IsCmdFiles(context.SelectedItem.FileExtension));
```
I think text files doesn't need this option as they are opened in notepad by default.

Reply:
I just came to the same realization!